packages/network-of-terms-catalog/catalog/datasets/aat-materials.jsonld

```diff
@@ -14,6 +14,11 @@
       "@value": "AAT - materialen"
     }
   ],
+  "genre": [
+    {
+      "@id": "https://data.cultureelerfgoed.nl/termennetwerk/onderwerpen/Abstracte-begrippen"
```

Comment:
@rschalkrce Please note that I just made this up. Could you please replace this with the real genre(s)?